www/providers.json

```diff
@@ -11,6 +11,7 @@
   "credentials": "Credentials",
   "discord": "Discord",
   "email": "Email",
+  "eveonline": "EVE Online",
```

Comment:
I saw this both as `EVEOnline` and `EVE Online`. Either change it here or everywhere else. I think it would be nice to keep it consistent. Do you agree?

Reply:
Yes, I'll change it.

Reply:
From what I can see on their website, it should be `EVE Online`, I guess.

Reply:
ok thanks.

However I also needed to add `it` to the jwt callback as well:

```js
callbacks: {
  jwt: async (token, user, account, profile, isNewUser) => {
    if (profile) {
      token = {
        ...token,
        id: profile.CharacterID,
      }
    } 
    return token;
  },
  session: async (session, token) => {
    if (token) {
      session.user.id = token.id;
    }
    return session;
  }
}
```

Now I see this in the browser for `session`:

```
{"user":{"name":"*********","email":null,"image":"https://image.eveonline.com/Character/*********_128.jpg","id":*********},"expires":"2021-03-03T22:41:25.212Z"}
```

This is exactly what I want, showing the character ID in the session :)